notebooks/WFC3/calwf3_v1.0_cte/calwf3_with_v1.0_PCTE.ipynb

```diff
@@ -21,23 +21,25 @@
     "- Compare v1.0 and v2.0 products. \n",
```

Comment:
<p>Thank you for your work. I have a suggestion to increase code readability: consider using more descriptive variable names instead of 'cl_inputs' and 'cl'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>